addons/crm/models/crm_lead.py

```diff
@@ -1182,7 +1197,34 @@ def get_import_templates(self):
     # ------------------------------------------------------------
     # PLS
     # ------------------------------------------------------------
-
+    # Predictive lead scoring is computing the lead probability, based on won and lost leads from the past
+    # Each won/lost lead increments a frequency table, where we store, for each field/value couple, the number of
+    # won and lost leads.
+    #   E.g. : A won lead from Belgium will increase the won count of the frequency country_id='Belgium' by 1.
+    # The frequencies are split by team_id, so each team has his own frequencies environment. (Team A doesn't impact B)
+    # There are two main ways to build the frequency table:
+    #   - Live Increment: At each Won/lost, we increment directly the frequencies based on the lead values.
+    #       Done right BEFORE writing the lead as won or lost.
+    #       We consider a lead that will be marked as won or lost.
+    #       Used during usage, to ensure frequency table is always up to date
+    #   - One shot Rebuild: empty the frequency table and rebuild it from scratch, based on every already won/lost leads
+    #       Done AFTER the lead has been won or lost.
```

Comment:
When reading, this sentence makes it feel like you live increment before writing, then rebuild the whole thing after writing.
I'm guessing it's not the case, so I would not put this sentence here.

Tell me if I missed something.